fpdf/fpdf.py

```diff
@@ -52,7 +52,7 @@
 HERE = Path(__file__).resolve().parent
 
 # Global variables
-FPDF_VERSION = "2.3.0"
+FPDF_VERSION = "2.3.1"
```

Comment:
Please do not bump the lib version number as part of a PR.
Maintainers will increment it on the `master` branch when a new release will be performed.